guidelines/index.html

```diff
@@ -596,6 +596,14 @@
 					<p>When an authenticated session expires, the user can continue the activity without loss of data after re-authenticating.</p>
 				</section>
 			</section>
+      
+      <section class="sc">
+					<h4>Animation from interactions</h4>
+					<p class="conformance-level">A</p>
+					<p class="change">Unchanged</p>
+					<p>For significant animation triggered by a user action that is not an essential part of the action, there is a mechanism for the user to pause, stop or hide the animation whilst still performing the same action.</p>
```

Comment:
Hi @patrickhlauke ,

Thank you very much for the text and your comment on the [February 7 2017 survey](https://www.w3.org/2002/09/wbs/35422/SC_20170207/results#xq4).

I updated the text at [the top of this page](https://github.com/w3c/wcag21/pull/96#issue-204047227). As soon as we get the final wording settled and I figure out how to update the pull request in GitHub and I'll do that too :-)